presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -0,0 +1,258 @@
+===============
+Presto Verifier
+===============
+
+Presto Verifier is a tool to run queries and verify correctness. It can be used to test whether a
+new Presto version produce the correct query results, or to test if pairs of Presto queries have
+the same semantics.
+
+During each Presto release, Verifier is ran to ensure that there is no correctness regression.
+
+Using Verifier
+--------------
+
+In a MySQL database, create the following table and load it with the queries you would like to run:
+
+.. code-block:: sql
+
+    CREATE TABLE verifier_queries (
+        id int(11) unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+        suite varchar(256) NOT NULL,
+        name varchar(256) DEFAULT NULL,
+        control_catalog varchar(256) NOT NULL,
+        control_schema varchar(256) NOT NULL,
+        control_query text NOT NULL,
+        control_username varchar(256) DEFAULT NULL,
+        control_password varchar(256) DEFAULT NULL,
+        control_session_properties text DEFAULT NULL,
+        test_catalog varchar(256) NOT NULL,
+        test_schema varchar(256) NOT NULL,
+        test_query text NOT NULL,
+        test_username varchar(256) DEFAULT NULL,
+        test_password varchar(256) DEFAULT NULL,
+        test_session_properties text DEFAULT NULL)
+
+Next, create a ``config.properties`` file:
+
+.. code-block:: none
+
+    source-query.suite=my_suite
+    source-query.database=jdbc:mysql://localhost:3306/my_database?user=my_username&password=my_password
+    control.gateway=jdbc:presto://localhost:8080
+    test.gateway=jdbc:presto://localhost:8081
+    test-id=1
+
+Download :maven_download:`verifier` and rename it to ``verifier``. To run the Verifier:
+
+.. code-block:: none
+
+    chmod +x verifier
+    ./verifier verify config.properties
+
+
+Verifier Procedures
+-------------------
+
+The following steps summarize the workflow of Verifier.
```

Comment:
I think using the imperative mood (read, apply, rewrite) for these descriptions is a bit confusing because it's not clear when you're instructing the user to do something vs. when you're just describing what the verifier does. It would be helpful if when you're describing what the verifier does you just used present tense (e.g. Reads the list of source queries, Applies overrides to the catalog, ...), and only use imperative if you're instructing the user to do something (e.g. set up the mysql database)